src/libs/ComponentSpecs.js

```diff
@@ -266,6 +266,10 @@ const prototypeSpecs = {
       const domNode = reactI13n.getScrollableContainerDOMNode();
       if (domNode) {
         options.target = domNode;
+        if (container){
+            options.touchMoveEndTarget=container;
+         
+        }
```

Comment:
1. how is `container` defined?
2. we might want to make the code more generic later, meaning, not specific to `touchMoveEnd` event. it's probably ok for now, since this is only for viewport detection...
3. code style: remove blank line on line 271 and add spaces between `){` and around `=`

Reply:
I've added those changes. Unfortunately I cannot test fully locally as I wasn't able to get it running locally properly but I did test by copying the lines changed to node modules in yahoo news so tested the functionality there.